### DISCUSSION TITLE
Station Offline Message, Do not have current song, artist listed.  There is no history of songs listed

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#6291] [Q&A]
![image](https://github.com/AzuraCast/AzuraCast/assets/132607615/25197040-228d-45a4-861d-2e07fb713d2f)

I can press play and get the music playing but current song list, artist, song name are missing.  Message says station is offline

**Comment by BusterNeece** [comment]
This should be an Issue, not a Discussion, and should follow our template for new issues.
